bin/p2-launch/main.go

```diff
@@ -36,3 +53,44 @@ func main() {
 		log.Fatalln("Unsuccessful launch of one or more things in the manifest")
 	}
 }
+
+func authorize(manifest pods.Manifest) error {
+	var policy auth.Policy
+	var err error
+	switch *authType {
+	case auth.NULL_POLICY:
+		return nil
+	case auth.KEYRING_POLICY:
+		return util.Errorf("--auth-type of '%s' currently not supported by p2-launch", *authType)
```

Comment:
i didn't support it because you have to provide a list of authorized users and wasn't sure how best to do that from the command line